### PR TITLE
feat: hubspot custom fields api (no picklist)

### PR DESCRIPTION
**Pull request by asdfryan**
Tested locally

### CODE REVIEW

**Comment by tomkit** [comment]
I'd like to do a play-through test of this on staging before release

**Comment by asdfryan** [comment]
> I'd like to do a play-through test of this on staging before release

That sounds fine. Could you review this PR so I can land to staging?

File: packages/core/remotes/impl/hubspot/index.ts [hunk 1]
```diff
@@ -1213,6 +1222,86 @@ class HubSpotClient extends AbstractCrmRemoteClient implements MarketingAutomati
...
+    await retryWhenRateLimited(async () => {
+      await this.maybeRefreshAccessToken();
+      try {
+        await this.#client.crm.properties.groupsApi.create(objectName, {
+          name: groupName,
+          label: groupName,
+        });
+      } catch (e) {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```
Comment (lucasmarshall) [inline packages/core/remotes/impl/hubspot/index.ts:1295]: Why are we using `@ts-ignore`?

Comment (asdfryan) [reply packages/core/remotes/impl/hubspot/index.ts:1295]: Switched to use any

**Review by tomkit — APPROVED** [review]

**Pull request merged**